src/components/seeds/database/filters/FilterCountWeight.tsx

```diff
@@ -177,32 +177,16 @@ export default function FilterCountWeight(props: Props): JSX.Element {
           />
         </Grid>
         <Grid item xs={5}>
-          <TextField
-            id='countMinValue'
-            value={countMinValue}
-            onChange={(value) => onChange('countMinValue', value)}
-            label={strings.MIN}
-          />
+          <TextField id='countMinValue' value={countMinValue} onChange={onChange} label={strings.MIN} />
```

Comment:
did we miss updating this signature in the onChange(id,value) cleanup?

Reply:
nvm, i think we should use the Textfield from web-components.. why this one left over?

Reply:
@charlie-tf just confirming.. the filters epic will be redoing these filters,, yes?

Reply:
Hey Karthik. Yes, the seeds database filters are to be largely revamped

Reply:
I don't think this bug was introduced with the onChange pr, because these textfields, as you said, are not from web-components. We can delete all this after the new implementation